proto/osmosis/twap/v1beta1/query.yml

```diff
@@ -0,0 +1,12 @@
+keeper: 
+  path: "github.com/osmosis-labs/osmosis/v10/x/twap"
```

Comment:
Can we also update our import paths job to replace the version here please:
https://github.com/osmosis-labs/osmosis/blob/main/scripts/replace_import_paths.sh

Reply:
Why is that not just a sed over every single file?